src/items/itemDialog.component.test.tsx

```diff
@@ -200,6 +200,45 @@ describe('ItemDialog', () => {
       });
     });
 
+    it('adds an item where the item property has an allowed list of values', async () => {
```

Comment:
Could this also have a string allowed_values list instead of only numbers, there are a few others as well.